### PR TITLE
Copy edits

### PR DESCRIPTION
**Pull request by xybrnet**

### CODE REVIEW

File: wazimap_za/profiles/ecd.py [hunk 1]
```diff
@@ -373,15 +367,15 @@ def get_ecd_budgets_profile(geo_code, geo_level, session):
         geo_level, geo_code, percent=False)
 
     csg = ecd_grants['child_support_grant']['values']['this']
-    monthly_csg = 330.00
+    monthly_csg = 350.00
```
Comment (longhotsummer) [inline wazimap_za/profiles/ecd.py]: Where's this from?

**Comment by longhotsummer** [comment]
Nice.

File: wazimap_za/profiles/ecd.py (hunk 1 above)
Comment (xybrnet) [inline wazimap_za/profiles/ecd.py]: It's from here: http://www.gov.za/services/child-care-social-benefits/child-support-grant
William sent it to us when we discussed adding this initially.

Comment (longhotsummer) [inline wazimap_za/profiles/ecd.py]: Oh ok. Perhaps put a comment in the code pointing at that? It'll save our future selves some confusion :)

**Pull request merged**